package.json

```diff
@@ -156,7 +156,8 @@
     "test:e2e:2": "CYPRESS_TEST_FILES=2*.js npm run cypress:run",
     "test:e2e:3": "CYPRESS_TEST_FILES=3*.js npm run cypress:run",
     "test:jest": "cross-env TZ=UTC jest components lib pages",
-    "test:update": "npm run test:jest -- --updateSnapshot"
+    "test:update": "npm run test:jest -- --updateSnapshot",
+    "lighthouse": "lhci autorun"
```

Comment:
Maybe not useful if it's not available as devDependency.

Reply:
Done.